README.md

```diff
@@ -152,7 +152,7 @@ lwip.open('image.jpg', function(err, image){
 - GIF (no animations)
 
 Other formats may also be supported in the future, but are probably less urgent.
-Check the issues to see [which formats are planned to be supported](https://github.com/EyalAr/lwip/issues?labels=format+request&page=1&state=open).
+Check the issues to see [which formats may be supported in the future](https://github.com/EyalAr/lwip/issues?labels=format+request&page=1&state=open).
```

Comment:
This phrasing implies that a certain format listed in the issues may never be supported. But if it's in the issues list, I do intend to support it at one point.
I prefer the old phrasing.